src/app/hooks/queries/useConfirmedBtcBalance.ts

```diff
@@ -0,0 +1,30 @@
+import useBtcClient from '@hooks/useBtcClient';
+import { useQuery } from '@tanstack/react-query';
+import useWalletSelector from '../useWalletSelector';
+
+const useConfirmBtcBalance = () => {
+  const { btcAddress } = useWalletSelector();
+  const btcClient = useBtcClient();
+
+  const fetchBtcAddressData = async () => btcClient.getAddressData(btcAddress);
+
+  let confirmedBalance: number = 0;
```

Comment:
```suggestion
  let confirmedBalance = 0;
```

Reply:
nit: do we need to explicitly set the type here? I think we can rely on type inference here https://www.typescriptlang.org/docs/handbook/type-inference.html